src/pages/checkout/index.js

```diff
@@ -1,13 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import _, { sum } from 'lodash'
+import './checkout.css'
+import {BasketContext}  from '../../context/basket'
+import ItemList from '../home/items-list'
+import Input from '../../components/input/input'
+import Button from '../../components/button/button'
 
 
 
-function App() {
+
+const Checkout = (props) => {
+  
+  const basket = useContext(BasketContext)
+  
+  
+  const [address, setAddress] = useState ('')
+  const [postCode, setPostCode] = useState ('')
+  
+  const onClick = () => {
+    props.history.push('/confirmation')
+  }
+  const submitter = () => {
+    return (
+      <div>
+       {address.length === 0, postCode.length===0 ? null : onClick()}
```

Comment:
Does this work? never seem a , in a ternary

Reply:
yep seems to work fine, I used it on the sign up page too and it seems to prevent the button from working unless both inputs contain text. happy to change to a different method though!

Reply:
If it works it's ok for now we can make it better later
I would have though that `address.length && postCode.length` would do the same thing though